recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
   run:
     - python
     - cloudpickle >=1.6.0,<=3.0.0,!=2.1.0,!=2.2.0
-    - snowflake-connector-python >=3.14.0,<4.0.0
+    - snowflake-connector-python >=3.12.0,<4.0.0
```

Comment:
why is it changed back to 3.12?

Reply:
https://github.com/snowflakedb/snowpark-python/pull/3119#discussion_r1982349309

Reply:
since 3.14.0 is not in conda, it is causing build failures.

Reply:
got it